report/main.qmd

```diff
@@ -1 +1,166 @@
+
+## EDA Analysis
+Before doing any analysis, it is important to first assess data missingness. A graph (like Figure 1) can be drawn to visualize the proportion of missing values for all columns in the dataset. If there are missing data, it is suggested to create a table to show the data missingness by `Site` for columns that have missing data (see Table 1). To see for each site from which intervention type the missing data comes from, an expanded table can be made to show the data missingness by `Site` and `Intervention` for columns with missing data (see Table A1 in Appendix). For columns that will be used in the analysis to answer the research question, it is important to find out why the data are missing.
+\
+```{r}
+sale_data <- read.table(here("rawdata/june1data.csv"), head=T, sep=",")
+
+# replace "dis " with "dis"
+sale_data <- sale_data %>%
+  mutate(Intervention = replace(Intervention, Intervention == "dis ", "dis"))
+
+#convert DofW, Site and Intervention columns to factor
+sale_data <- sale_data %>% 
+  mutate_at(c('DofW', 'Site', 'Intervention'), as.factor)
+```
+
+```{r missing-data-plot, fig.height=3}
+#missing data, overall
+plot_missing_values_proportions(df=sale_data) +
+  ggtitle("", subtitle = "Figure 1: The proportion of missing values for all columns in \nthe dataset.")
+```
+
+```{r missing-data-table}
+# data missingness  by Site
+count_missing<- sale_data %>%
+    group_by(Site) %>%
+    summarise(across(.cols = where(~ any(is.na(.x))), ~sum(is.na(.x))))
+
+# Table
+kable(count_missing, caption = "Number of missing observations for columns with missing data present by site")
+```
+
+To explore the change of daily sales of zero-calorie beverages and bottled sugared beverages over time, time series plot for `ZeroCal` and `Sugary` can be made for each of the three sites. It is important to note that the increase in the raw counts of say zero-calorie beverages being sold does not necessarily mean the decrease in the sales of sugared beverages. It could be the case that more customs came to the shops that day so that sales of all things tended to be higher. Therefore, in addition to visualize the change of daily sales using the raw counts, it is even more important to explore the change of daily sales in percentage terms (i.e. `ZeroCal`/(`ZeroCal`+`Sugary`)\*100\% and `Sugary`/(`ZeroCal`+`Sugary`)\*100\%)
```

Comment:
"time series plots" or "a time series plot"
"a decrease in sugared beverage sales"  instead of "the decrease in the sales of sugared beverages"
"customers" not "customs'
"so all sales" instead of "so that sales of all things"
"visualizing", not "visualize"